libbeat/scripts/generate_fields_docs.py

```diff
@@ -136,7 +138,8 @@ def fields_to_asciidoc(input, output, beat):
         if "anchor" not in section:
             section["anchor"] = section["key"]
 
-        output.write("* <<exported-fields-{}>>\n".format(section["anchor"]))
+        if  "nofields" not in section:
```

Comment:
The sections are collected in the for loop line 135. You could filter out sections early.

Reply:
Hmm....I dunno. When I filter out sections earlier, it removes sections that have valid fields.  The logic here is a bit hard for me to follow because of how the script loops over the nested sections in the YAML file. Can you be more specific about what I need to add and where?